### PR TITLE
Merge to "master" from development

### PR DESCRIPTION
**Pull request by rajeevbbqq**

### CODE REVIEW

File: airdrop/config.py [hunk 1]
```diff
@@ -24,6 +24,11 @@
 SIGNER_PRIVATE_KEY = 'AIRDROP_SIGNER_PRIVATE_KEY'
 SIGNER_PRIVATE_KEY_STORAGE_REGION = ''
 
+NUNET_TOKEN_ADDRESS = ''
```
Comment (prashantramangupta) [inline airdrop/config.py:27]: As decided token address and token name needs to be part of database

File: airdrop/infrastructure/models.py [hunk 2]
```diff
@@ -164,6 +164,7 @@ class ClaimHistory(Base, AuditClass):
         nullable=False,
     )
     address = Column("address", VARCHAR(50), nullable=False, index=True)
+    type = Column("type", VARCHAR(50), nullable=True)
```
Comment (prashantramangupta) [inline airdrop/infrastructure/models.py:167]: column name type is very generic. can we have a more meaningful column name here?

File: airdrop/serverless.yml [hunk 3]
```diff
@@ -417,6 +417,51 @@ functions:
...
+      securityGroupIds:
+        - ${file(./config.${self:provider.stage}.json):SG1}
+        - ${file(./config.${self:provider.stage}.json):SG2}
+      subnetIds:
+        - ${file(./config.${self:provider.stage}.json):VPC1}
+        - ${file(./config.${self:provider.stage}.json):VPC2}
+    events:
+      - http:
+          path: /airdrop-window-stake-details
+          method: post
```
Comment (prashantramangupta) [inline airdrop/serverless.yml:434]: Why do we need the Post method for "get" calls?

File: airdrop/infrastructure/repositories/airdrop_repository.py [hunk 4]
```diff
@@ -217,4 +225,4 @@ def get_airdrops_schedule(self, token_address):
         if airdrop_row_data is not None:
             return AirdropFactory.convert_airdrop_schedule_model_to_entity_model(
                 airdrop_row_data)
-        raise Exception("Invalid token address")
+        raise Exception("Invalid airdrop id")
```
Comment (prashantramangupta) [inline airdrop/infrastructure/repositories/airdrop_repository.py:228]: Let's have a custom exception here.

File: airdrop/infrastructure/repositories/airdrop_repository.py [hunk 5]
```diff
@@ -151,13 +151,21 @@ def register_airdrop_window_timeline(self, airdrop_window_id, title, description
             airdrop_window_id=airdrop_window_id, title=title, description=description, date=date)
         return self.add(airdrop_window_timeline)
 
+    def get_contract_address(self, airdrop_id):
```
Comment (prashantramangupta) [inline airdrop/infrastructure/repositories/airdrop_repository.py:154]: Let's have a separate file for blockchain utility methods.

**Review by anandrgitnirman — APPROVED** [review]
Based on testing done in Ropsten

**Pull request merged**